### PR TITLE
feature/WAL-586

### PR DESCRIPTION
**Pull request by sksadjad**

### CODE REVIEW

File: src/services/qrService.ts [hunk 1]
```diff
@@ -380,7 +381,7 @@ const connectOpenId4VcIssuance = async (args: IQrDataArgs): Promise<void> => {
     }).then((contacts: Array<IContact>) => {
       if (contacts.length === 0) {
         args.navigation.navigate(ScreenRoutesEnum.CONTACT_ADD, {
-          name: url.hostname,
+          name,
```
Comment (nklomp) [inline src/services/qrService.ts:384]: name || url.hostname

**Review by nklomp — COMMENTED** [review]
should fallback to url as mentioned

File: src/services/qrService.ts (hunk 1 above)
Comment (sksadjad) [reply src/services/qrService.ts:384]: I'm doing exactly that a few lines above 370

Comment (nklomp) [reply src/services/qrService.ts:384]: Oh okay, because of the long line in Github didn't see that

**Review by nklomp — APPROVED** [review]

**Pull request merged**